### ISSUE TITLE
Unit tests fail sometimes

### ISSUE THREAD

**Issue by samreid** [phetsims/balloons-and-static-electricity#434]
I ran BASE unit tests about 20 times this morning and had 3 failures.  The URL is http://localhost/balloons-and-static-electricity/balloons-and-static-electricity-tests.html?ea . Looks like it may be related to accessibility.

**Comment by jessegreenberg** [comment]
Thanks, I think all tests in this sim are related to a11y. Ill take a look.

**Comment by jessegreenberg** [comment]
I ran the tests about 20 times and didn't encounter an error. Were tests run in a background window? Maybe if tests are using DOM `focus` things are failing while the test is run in the background. Ill look.

**Comment by jessegreenberg** [comment]
I don't see that happening in tests. None of the tests mimic user input they all go through the model.

@samreid did you run this test in a new tool or did you just reload the provided link? If through a new tool, is there a way I can run it?

**Comment by samreid** [comment]
@jessegreenberg and I worked together to convert toImage to rasterized and it is working nicely so far.  The layout didn't match the prior layout exactly, and @jessegreenberg volunteered to investigate further.  Thanks!

**Comment by jessegreenberg** [comment]
I think I figured out the cause for difference in layout, before the change to use `rasterized`, we were using toImage without specifying `x`, `y`, `width`, `height`. When these aren't used, `toCanvas` (used by `toImage`) adds extra padding to the canvas:
```js
      var padding = 2; // padding used if x and y are not set

      // ....

      x = x !== undefined ? x : Math.ceil( padding - bounds.minX );
      y = y !== undefined ? y : Math.ceil( padding - bounds.minY );
      width = width !== undefined ? width : Math.ceil( bounds.getWidth() + 2 * padding );
      height = height !== undefined ? height : Math.ceil( bounds.getHeight() + 2 * padding );

      var canvas = document.createElement( 'canvas' );
      canvas.width = width;
      canvas.height = height;
```

So If I add 1 to the translation in x and y it looks correct.

**Comment by jessegreenberg** [comment]
Instead of adding padding to the translation maybe I can use `sourceBounds` option of rasterized to match the layout.

**Comment by jessegreenberg** [comment]
OK, I haven't heard about this recently (granted #438 could have been hiding this) - but I think this is safe to close. If it shows up again we can reopen. Closing.
